translations/messages+intl-icu.de.yml

```diff
@@ -1029,6 +1029,7 @@ error.copy.mapfile: "Die Einzeichnung der Stellungnahme {externId} konnte nicht
 error.changes.not.saved: "Beim Speichern der Änderungen ist ein Fehler aufgetreten."
 error.date.authored: "Bitte geben Sie ein Verfassungsdatum ein."
 error.in.fields: "Bitte korrigieren Sie Ihre Eingabe in folgenden Feldern: "
+error.phone.required: "Bitte geben Sie eine Telefonnummer ein."
 error.phone_or_email.required: "Bitte geben Sie entweder eine gültige E-Mail-Adresse oder Telefonnummer ein."
 error.phone.pattern: "Bitte geben Sie gültige Telefonnummer ein."
```

Comment:
nitpick: not in alphabetic sort order

Reply:
done https://github.com/demos-europe/demosplan-core/pull/2156/commits/a608bf8147a90024fda1261b1937f04e41da45eb